src/types/course.d.ts

```diff
@@ -78,6 +78,10 @@ export type Challenge = {
   teamLimit?: number;
   isHackathon?: boolean;
   multipleSubmissions?: boolean;
+  additionalInfo?: {
+    gradingCriteria?: string;
+    teamFormation?: string;
+  };
```

Comment:
I suggest the following type for additional information.

```ts
type Challenge = {
    // ... other challenge attributes
    additionInfos: {[type: string]: AdditionInfo};
}


type AdditionInfo = {
    title?: string;
    text: string;
}


const challenge = {
    additionInfos: {
        "GRADING_CRITERIA": {
            text: "The grading criteria is based on the following aspects: ..."
        },
        "TEAM_FORMATION": {
            text: "This is an example of how to solve the challenge."
        },
        "TIPS": {
            text: "Here are some tips to solve the challenge."
        }
    }
}
```

I would also store these additional info types as constants, mainly if used in multiple places.

eg:

```ts
// constants.ts

export const GRADING_CRITERIA = "GRADING_CRITERIA";
export const TEAM_FORMATION = "TEAM_FORMATION";
```

usage

```ts
const gradingInfo = challenge.additionalInfo[GRADING_CRITERIA];
```